manifest.jsonnet

```diff
@@ -46,7 +46,7 @@ else
 
 local INDEX_MANAGER_FILES = ['core/storage.js', 'index-manager.js'];
 json.addIcons(icons())
-.addPermissions(['storage', 'unlimitedStorage'])
+.addPermissions(['storage', 'unlimitedStorage', '*://crates.io/api/v1/crates/*'])
```

Comment:
We can remove `*://crates.io/api/v1/crates/*` too.
> Permission '*://crates.io/api/v1/crates/*' is unknown or URL pattern is malformed.

Reply:
Without this, Firefox says `TypeError: NetworkError when attempting to fetch resource.`

Reply:
I found this, will make an update.
https://www.extension.ninja/blog/post/solved-permission-is-unknown-or-url-pattern-is-malformed/

Reply:
Good to know. Thanks.